src/css/global/fonts.scss

```diff
@@ -15,11 +15,11 @@
 
 @font-face {
   font-family: 'hylo-evo-icons';
-  src:  url('/assets/fonts/icons/hylo-evo-icons.eot');
-  src:  url('/assets/fonts/icons/hylo-evo-icons.eot#iefix') format('embedded-opentype'),
-    url('/assets/fonts/icons/hylo-evo-icons.ttf') format('truetype'),
-    url('/assets/fonts/icons/hylo-evo-icons.woff') format('woff'),
-    url('/assets/fonts/icons/hylo-evo-icons.svg#hylo-evo-icons') format('svg');
+  src:  url('/assets/fonts/icons/hylo-evo-icons.eot?ne5jg1');
```

Comment:
ooh, good call, we should always do this to cache bust

Reply:
Hmm... This is handled automatically by the CSS/PostCSS loader through the WebPack build and if for some reason that is not working should be remedied.

Reply:
Ok took it out in 46e50383